src/bc5.rs

```diff
@@ -8,37 +8,24 @@ pub fn calc_output_size(width: u32, height: u32) -> usize {
     block_count * 16
```

Comment:
Question: is the output size calculation still correct, given that you now use less bytes - channels even - per pixel?